src/components/SearchComponent.tsx

```diff
@@ -133,7 +133,7 @@ const SearchComponent: React.FC<SearchComponentProps> = ({
                   <LinkWithNewTabHandling
                     href={
                       shouldOpenInEditor
-                        ? `/editor?snippet_id=${snippet.snippet_id}`
+                        ? `/editor?package_id=${snippet.snippet_id}`
```

Comment:
```suggestion
                        ? `/editor?package_id=${package.package_id}`
```

I think you should do a full refactor rather than just updating the path name